go.mod

```diff
@@ -74,3 +74,5 @@ require (
 require github.com/docker/libnetwork v0.8.0-dev.2.0.20190625141545-5a177b73e316 // indirect
 
 replace github.com/opencontainers/runc => github.com/opencontainers/runc v1.1.1-0.20220617142545-8b9452f75cbc
+
+replace github.com/containers/common => github.com/karthikelango137/common v0.48.1-0.20220727150028-71a5d2c2db12
```

Comment:
Just to be extra sure.  Before merging, we need a PR in c/common and then vendor from there.